pkg/conversions/gatewayroute_types_conversion_test.go

```diff
@@ -372,6 +373,114 @@ func TestConvert_CRD_GRPCGatewayRouteMatch_To_SDK_GrpcGatewayRouteMatch(t *testi
 	}
 }
 
+func TestConvert_CRD_GRPCHostnameRewrite_To_SDK_GrpcHostnameRewrite(t *testing.T) {
+	type args struct {
+		crdObj *appmesh.GrpcGatewayRouteRewrite
+		sdkObj *appmeshsdk.GrpcGatewayRouteRewrite
+	}
+	tests := []struct {
+		name       string
+		args       args
+		wantSDKObj *appmeshsdk.GrpcGatewayRouteRewrite
+	}{
+		{
+			name: "Valid Case",
+			args: args{
+				crdObj: &appmesh.GrpcGatewayRouteRewrite{
+					Hostname: &appmesh.GatewayRouteHostnameRewrite{
+						DefaultTargetHostname: aws.String(string("enabled")),
+					},
+				},
+				sdkObj: &appmeshsdk.GrpcGatewayRouteRewrite{},
+			},
+			wantSDKObj: &appmeshsdk.GrpcGatewayRouteRewrite{
+				Hostname: &appmeshsdk.GatewayRouteHostnameRewrite{
+					DefaultTargetHostname: aws.String(string("enabled")),
+				},
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			Convert_CRD_GRPCHostnameRewrite_To_SDK_GrpcHostnameRewrite(tt.args.crdObj, tt.args.sdkObj)
+			assert.Equal(t, tt.wantSDKObj, tt.args.sdkObj)
+		})
+	}
+}
+
+func TestConvert_CRD_HTTPGatewayRoutePathRewrite_To_SDK_HttpGatewayRoutePathRewrite(t *testing.T) {
+	type args struct {
+		crdObj *appmesh.HTTPGatewayRouteRewrite
+		sdkObj *appmeshsdk.HttpGatewayRouteRewrite
+	}
+	tests := []struct {
+		name       string
+		args       args
+		wantSDKObj *appmeshsdk.HttpGatewayRouteRewrite
+	}{
+		{
```

Comment:
We need to add test cases for all the options we support in the code (i.e.,) Exact/Regex etc. Otherwise we will not exercise those code flows as part of this UT.

Reply:
Also, cases where multiple options are specified and verify the error msg that we bubble up if those are not supported.

Reply:
I don't see the new test cases, yet...Are they located elsewhere?

Reply:
wil add them